packages/evolution-frontend/src/components/inputs/InputRadioNumber.tsx

```diff
@@ -79,21 +80,18 @@ const InputRadioNumberChoice = ({
     );
 };
 
-export const InputRadioNumber = ({ id, onValueChange, value, widgetConfig, interview, path, user }: InputRadioNumberProps<any, any, any, any>) => {
-    const minValue =
-        (surveyHelper.parseInteger(
-            widgetConfig.valueRange.min,
-            interview,
-            path,
-            user
-        ) || 0);
-    const maxValue =
-        (surveyHelper.parseInteger(
-            widgetConfig.valueRange.max,
-            interview,
-            path,
-            user
-        ) || minValue + 1 );
+export const InputRadioNumber = <CustomSurvey, CustomHousehold, CustomHome, CustomPerson>({
+    id,
+    onValueChange,
+    value,
+    widgetConfig,
+    interview,
+    path,
+    user,
+    t
+}: InputRadioNumberProps<CustomSurvey, CustomHousehold, CustomHome, CustomPerson> & WithTranslation) => {
```

Comment:
You should wrap this widget in the `withTranslation` hoc, otherwise it does not compile. See InputTime.tsx, the last line